runbooks/how-to-update-for-rda-api-changes.md

```diff
@@ -0,0 +1,53 @@
+# How to Update for RDA API Changes
+
+1. Synch RDA API changes
+   1. Clone/pull latest changes from RDA API repo
+       - https://github.cms.gov/medicare-payment-modernization/rda-api
+   2. Copy changes from RDA API proto files to BFD proto files
+        - RDA API: https://github.cms.gov/medicare-payment-modernization/rda-api/tree/main/proto/src/main/proto/rda/v1
+        - BFD: https://github.cms.gov/medicare-payment-modernization/rda-api/tree/main/proto/src/main/proto/rda/v1
```

Comment:
Add the releases page in the RDA API repo and mention they can just download the zip for the release they need.